### PR TITLE
jenkins/cloud: Retain AMI in images directory

### PR DESCRIPTION
**Pull request by cgwalters**
So anyone can upload it to another account easily.

The downside of this is the images directory gets larger, but oh well.

### CODE REVIEW

**Comment by cgwalters** [comment]
Not tested yet though!

File: Jenkinsfile.cloud [hunk 1]
```diff
@@ -153,6 +153,7 @@ node(NODE) {
         img_prefix = "${dirpath}/rhcos-${version}"
         qcow = "${img_prefix}.qcow2"
         vmdk = "${img_prefix}.vmdk"
+        ec2 = "${img_prefix}.ami"
```
Comment (dustymabe) [inline Jenkinsfile.cloud]: dumb question.. where does this variable get used?

Comment (dustymabe) [reply Jenkinsfile.cloud]: ahh looks like line 172

**Comment by dustymabe** [comment]
LGTM. I'll let @cgwalters merge once he thinks testing is sufficient

**Comment by ashcrow** [comment]
Needs a rebase.

**Comment by cgwalters** [comment]
I don't think we need this one now that the AMI is public, closing.

**Pull request closed without merging**